encoding/encode.go

```diff
@@ -18,36 +18,49 @@ package encoding
 
 import (
 	"encoding/hex"
+	"fmt"
 	"log"
-	"strconv"
 )
 
 // In koa, we use hexadecimal encoding
 
-// TODO: implement w/ test cases :-)
-func Encode(operand interface{}) []byte {
+// EncodeOperand() encodes operand to bytes.
+func EncodeOperand(operand interface{}) []byte {
 	switch op := operand.(type) {
 	case int:
-		return nil
+		i, err := encodeInt(op)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		return i
+
```

Comment:
It would be better to hand it over to return rather than to handle the error here.
```
func EncodeOperand(operand interface{}) ([]byte,error){
    return encodeInt(op)
}
```